src/index.html

```diff
@@ -1,13 +1,103 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="main-container">
+      <div data-qa="card" class="container__box-card">
+
+        <div class="img-container">
+          <img
+       class="img-container__picture"
+       src="./images/imac.jpeg"
+       alt="i-mac">
+        </div>
+
+        <div class="prod-discribe">
+      Моноблок APPLE A1419 iMac<br> 27" Retina 5K (MNED2UA/A)
+          <div class="prod-discribe__code">Код товара: 195434
+          </div>
+        </div>
+
+        <div class="rating-container">
+          <div class="rating-container__stars">
+            <div class="stars"></div>
+            <div class="stars"></div>
+            <div class="stars"></div>
+            <div class="stars"></div>
+            <div class="stars"></div>
+          </div>
+          <div class="rating-container__feedback-container">
+            <div class="feedback">Отзывов: 5</div>
+          </div>
+        </div>
+
+
+        <div class="price-box">
+          <div class="price-box__cena">Цена:</div>
+          <div class="price-box__price">69 999 грн</div>
+        </div>
+
+        <div class="button-container">
+          <a class="button-container__link" href="http://google.com">
+            <div data-qa="hover" class="button-container__button">
+          КУПИТЬ
+            </div>
+          </a>
+        </div>
+      </div>
+
+      <div data-qa="card" class="container__box-card">
+
+        <div class="img-container">
+          <img
+       class="img-container__picture"
+       src="./images/imac.jpeg"
+       alt="i-mac">
+        </div>
```

Comment:
```suggestion
          <img
            class="img-container__picture"
            src="./images/imac.jpeg"
            alt="i-mac"
          >
        </div>
```
Поправь везде отступы